cluster/examples/database/mysqlinstance/azure/resource-class.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 specTemplate:
   forProvider:
     administratorLogin: myadmin
-    resourceGroupName: muvaf-group-westus-1
+    resourceGroupName: example-group
```

Comment:
looks like this group name used to be `group-westus-1`, do you want to restore it back to that? (or does it not matter)

https://github.com/crossplaneio/crossplane/blob/b3e96eeae7fd9330a55e352ea7bee4fafec96eb4/cluster/examples/database/mysqlinstance/azure/resource-class.yaml#L13

Reply:
@jbw976 whatever group is listed here will have to have been previously created, so whatever we put here isn’t valid by default. In general, many resources are dependent on the existence (or absence) of other resources, so unless a resource is part of a full guide, we are only basically saying “this will not be rejected by the API server”